tests/model/test_model.py

```diff
@@ -628,6 +628,33 @@ def test_modernize_message_response(self, model, messages,
         assert (model.modernize_message_response(messages)
                 == expected_messages_response)
 
+    @pytest.mark.parametrize('feature_level, to_vary_in_initial_data', [
+        (None, {}),
+        (27, {}),
```

Comment:
A value between is good - I see that this is for 3.1, though I wouldn't know from reading the code.

Of course here we're defining the input *plus* the feature level. Like you said elsewhere, we could even skip the feature level since we don't rely on it right now - though it's useful to include for documentation? If we do include an extra feature level I'd also/instead suggest the level just before the change.

Reply:
I initially tried to keep the fixtures matching with the server versions that I have tested locally. :)

Re skipping feature level, I think we should keep that since it indicates that these fields are actually dependent on the server version, and adding a feature level just before the change would also indicate precisely when these fields were added in the server.